src/content/HighAffinity.tsx

```diff
@@ -6,11 +6,45 @@ import Definition from "../components/shared/Definition";
 import { PageContent, Section, LayoutType } from "../types";
 
 export const highAffinityContentArray: PageContent[] = [
-    // making the content array 1 indexed to match the page numbers
     {
-        content: "",
-        section: Section.Introduction,
+        title: "Binding Affinity",
+        content: (
+            <>
+                <p>
+                    Welcome to the Binding Affinity interactive application!
+                    Here, you will learn how to determine the strength of
+                    interaction between two molecules in the lab by using
+                    computer simulations.{" "}
+                </p>
+                <p>
+                    This educational module is designed to be completed in a
+                    linear sequence, starting with high affinity, which contains
+                    important introductory content and guidance. However, if you
+                    complete the proceeding module, you may use the navigation
+                    at the top of the page to “drop in” on the next section.
+                </p>
+            </>
+        ),
+        section: Section.LandingPage,
         layout: LayoutType.FullScreenOverlay,
+        acknowledgment: (
+            <>
+                <div>
+                    David S. Goodsell, RCSB Protein Data Bank.
+                    doi:10.2210/rcsb_pdb/goodsell-gallery-021
+                </div>
+                <div>
+                    This painting is part of “VAX,” a series of paintings
```

Comment:
I _think_ when it's a title you can put the comma after the closed quote..

Reply:
oh you're right, I copied this text from his website